library/src/main/java/com/cottacush/android/currencyedittext/CurrencyInputWatcher.kt

```diff
@@ -97,7 +111,7 @@ class CurrencyInputWatcher(
             } else {
                 editText.setSelection(editText.text.length - 1)
             }
-        } catch (e: ParseException) {
+        } catch (e: Exception) {
```

Comment:
I know I was experiencing a crash before, with the "decimal separator before any number" use case, which was why i changed the to the generic `Exception`. But i checked again now and catching `Parse Exception` works again, so i'll change it back.
It may have been cause by the Big Decimal fix.